modules/rosa-create-dedicated-cluster-admins.adoc

```diff
@@ -4,8 +4,8 @@
 
 
 [id="rosa-create-dedicated-cluster-admins.adoc"]
```

Comment:
Not part of your change, but the ID above here shouldn't have a `.adoc` in it. E.g., should just be:

```
[id="rosa-create-dedicated-cluster-admins"]
```